core/base/src/TString.cxx

```diff
@@ -126,6 +126,11 @@ TString::TString(const std::string &s)
 
 TString::TString(const char *cs, Ssiz_t n)
 {
+   if (!cs) {
+      Error("TString::TString", "Missing input string!");
```

Comment:
```suggestion
      Error("TString::TString", "NULL input string!");
```
or similar? ("Missing" sounds like the argument was missing, but it's a nullptr)